packages/xrpl/test/integration/requests/accountTx.test.ts

```diff
@@ -32,6 +32,102 @@ describe('account_tx', function () {
         ledger_index: 'validated',
       }
       const response = await testContext.client.request(request)
+      const expected = {
+        result: {
+          account: testContext.wallet.classicAddress,
+          limit: 400,
+          transactions: [
+            {
+              tx_json: {
+                Account: 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh',
+                DeliverMax: '400000000',
+                Destination: testContext.wallet.classicAddress,
+                Fee: '12',
+                Flags: 0,
+                LastLedgerSequence: 1753,
+                Sequence: 843,
+                SigningPubKey:
+                  '0330E7FC9D56BB25D6893BA3F317AE5BCF33B3291BD63DB32654A313222F7FD020',
+                TransactionType: 'Payment',
+                TxnSignature:
+                  '30440220693D244BC13967E3DA67BDC974096784ED03DD4ACE6F36645E5176988452AFCF02200F8AB172432913899F27EC5523829AEDAD00CC2445690400E294EDF652A85945',
+                date: 685747005,
+                hash: '2E68BC15813B4A836FAC4D80E42E6FDA6410E99AB973937DEA5E6C2E9A116BAB',
+                ledger_index: 1734,
+              },
+            },
+          ],
+        },
+        type: 'response',
+      }
+      assert.equal(response.type, expected.type)
```

Comment:
could you include an `assert` for `tx_blob` and `meta_blob` fields? -- these would be visible only if "binary" format was requested. Nevertheless, these fields were newly introduced in V2.

I'm looking at the table here for reference: https://github.com/XRPLF/clio/issues/722

Reply:
How would a user request for `binary` format? I'm having trouble finding it.

Reply:
https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/account-methods/account_tx/
`"binary": true`

Reply:
Like I mentioned in https://github.com/XRPLF/xrpl.js/pull/2656#discussion_r1649162438 , I don't believe we should test for this specifically.

Reply:
okay yeah. The APIs have a wide surface area, its hard to cover every possibility